### ISSUE TITLE
Transcriptomics training 'RNAseq - counts to genes' - incorrect data in Shared Data

### ISSUE THREAD

**Issue by GraceAHall** [galaxyproject/training-material#2378]
Related tutorial: https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html

TLDR: Shared Data link to import files is incorrect, need to use provided zenodo links instead. Should alter the training material markdown. 

Under the 'Import Data' section, two options to upload files are given. By importing from Shared Data (GTN - Material -> transcriptomics -> 2: RNA-seq counts to genes), or by using provided zenodo links. 

The zenodo links contain the **_correct data_**, while the Shared Data is in a **_different format_**. Sample IDs are different in the Shared Data count matrix, and the factordata contains 4 cols, not 2. Additionally, there are 2 factordata files in the same folder, but have conflicting entries for some samples. 

Grace :)

**Comment by shiltemann** [comment]
Thanks @GraceAHall for letting us know! We will look into that :+1:
